.github/workflows/label_merge_conflicts.yml

```
@@ -2,7 +2,7 @@ name: 'Check for merge conflicts'
 on:
   schedule:
     # run this Action every 15 minutes
-    - cron:  '*/15 * * * *'
+    - cron:  '* */12 * * *'
```

```suggestion
    - cron:  '0 */12 * * *'
```
If there's a * in the first column, it'll fire every minute in the 0th and 12th hours.